tools/appliance/build.sh

```diff
@@ -235,7 +235,7 @@ function stage_vmx() {
 displayname = "${1}"
 annotation = "${1}"
 guestos = "otherlinux-64"
-virtualHW.version = "11"
+virtualHW.version = "13"
```

Comment:
will this be compatible with ESXi 6.x and 7.x @sureshanaparti cc @weizhouapache ?

Reply:
yes, compatible @rohityadavcloud 
https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vm-administration/GUID-64D4B1C9-CD5D-4C68-8B50-585F6A87EBA0.html)